paper/src/main/java/vg/civcraft/mc/civchat2/commands/GlobalChat.java

```diff
@@ -0,0 +1,73 @@
+package vg.civcraft.mc.civchat2.commands;
+
+import co.aikar.commands.BaseCommand;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandCompletion;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.Optional;
+import co.aikar.commands.annotation.Syntax;
+import org.bukkit.entity.Player;
+import vg.civcraft.mc.civchat2.ChatStrings;
+import vg.civcraft.mc.civchat2.CivChat2;
+import vg.civcraft.mc.civchat2.CivChat2Manager;
+import vg.civcraft.mc.namelayer.GroupManager;
+import vg.civcraft.mc.namelayer.NameAPI;
+import vg.civcraft.mc.namelayer.group.Group;
+import vg.civcraft.mc.namelayer.permission.PermissionType;
+
+public class GlobalChat extends BaseCommand {
+	@CommandAlias("global|globalchat")
+	@Syntax("[message]")
+	@Description("Enters global group chat or sends a message to global group chat")
+	@CommandCompletion("@CC_Groups @nothing")
+	
+	public void execute(Player player, @Optional String chatMessage) {
+		String globalGroupName = CivChat2.getInstance().getPluginConfig().getGlobalChatGroupName();
```

Comment:
Variables should be defined as closely to the point of their first usage as possible, the tigher the scope of a variable, the more understandable the code is.